pkg/dao/interfaces.go

```diff
@@ -11,3 +12,13 @@ type RepositoryDao interface {
 	List(orgID string, limit int, offset int) (api.RepositoryCollectionResponse, int64, error)
 	Delete(orgID string, uuid string) error
 }
+
+type RpmDao interface {
+	Create(orgID string, accountID string, repo *models.Repository, newRpm *models.Rpm) error
+	// TODO Implement
+	// Update(a *api.RepositoryRpm) error
+	Fetch(OrgID string, AccountID string, rpmUUID string) (*api.RepositoryRpm, error)
+	List(orgID string, accountID string, uuidRepo string, limit int, offset int) (api.RepositoryRpmCollectionResponse, int64, error)
+	// TODO Implement
+	// Delete(orgId string, accountNumber string, uuid string) error
```

Comment:
no need for update/delete here, if its not used in the code, don't include it

Reply:
thanks! updated